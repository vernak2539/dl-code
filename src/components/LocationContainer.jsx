import React from 'react';

export default ({ currentLocation }) => (
	<div className="logo-bg">
		<div className="section location-container">
			<div>
				<div className="location-label">Location</div>
				<div className="current-location">{currentLocation || 'Loading...'}</div>
			</div>
			<div>
				<button className="btn">Change Location</button>
			</div>
		</div>
	</div>
);
