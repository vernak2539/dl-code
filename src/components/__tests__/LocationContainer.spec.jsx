import React from 'react';
import LocationContainer from '../LocationContainer';
import { render } from 'enzyme';

describe('Component > Location Container', () => {
	it('should render correctly', () => {
		expect(render(<LocationContainer />)).toMatchSnapshot();
	});
});
