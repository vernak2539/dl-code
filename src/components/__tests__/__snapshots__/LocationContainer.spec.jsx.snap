// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > Location Container should render correctly 1`] = `
<div
  class="logo-bg"
>
  <div
    class="section location-container"
  >
    <div>
      <div
        class="location-label"
      >
        Location
      </div>
      <div
        class="current-location"
      >
        sample location
      </div>
    </div>
    <div>
      <button
        class="btn"
      >
        Change Location
      </button>
    </div>
  </div>
</div>
`;

exports[`Component > Location Container should render correctly when no location provided 1`] = `
<div
  class="logo-bg"
>
  <div
    class="section location-container"
  >
    <div>
      <div
        class="location-label"
      >
        Location
      </div>
      <div
        class="current-location"
      >
        Loading...
      </div>
    </div>
    <div>
      <button
        class="btn"
      >
        Change Location
      </button>
    </div>
  </div>
</div>
`;
